### ISSUE TITLE
[Bug]: a11y: The popover doesn't work correctly when the screen reader is on

### ISSUE THREAD

**Issue by orsolya-gyovai** [sbb-design-systems/lyne-components#4492]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Design Tokens

### Which version of Lyne Components are you using

4.4.0

### Operating system

Windows 11

### Browser / Browser version

Chrome (Version 145.0.7632.46)

### Input mode

Mouse/Keyboard

### What happened?

The tooltip is not displayed when the screen reader is switched on, it flickers. (The same thing happens with JAWS 2024 and in Edge, with both screen readers. In Firefox works everything fine)

https://github.com/user-attachments/assets/d91c338f-7c85-4a89-bc11-5f3345e46ae4

### Steps to reproduce the issue

1. Links: [https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-popover--without-close-button-hover](url)
[https://feature-sbbch-22341-lyne-4-update.dev-www.sbb.ch/de/angebote/swisstravelpass](url)
2. Open NVDA
3. Navigate with tha TAB key

### Relevant log output

```shell

```
